dhis-2/dhis-api/src/test/java/org/hisp/dhis/util/OrganisationUnitCriteriaUtilsTest.java

```diff
@@ -114,48 +101,4 @@ void testGetAnalyticsQueryCriteria_Aggregate() {
     // then
     assertEquals(validOuDimensions, analyticsQueryCriteria);
   }
-
```

Comment:
Can't these tests be adapted instead of removed? They are testing negative/invalid cases.

Reply:
those removed didn't make much sense. Created a bunch more to cover other cases